### DISCUSSION TITLE
[Enhancment] Divide links in About section to 2 lines (or 3)

### DISCUSSION THREAD

**Discussion by inson1** [hushenghao/AndroidEasterEggs#146] [Ideas]
Right now it looks like 1 line, that is divided by line break. 

I would divide it to 2 lines - 1. would be websites (Github, Translation), 2. would be actions (rest) (and maybe third from #142)

PS If you dont like the idea please move it to discussion

**Comment by hushenghao** [comment]
The About section is FlowGroup, Automatically wrap lines based on screen width

**Comment by hushenghao** [comment]
Outdated
